src/initializeGame.js

```diff
@@ -5,7 +5,7 @@ window.onload = function Initialize() {
     document.getElementById('fringeCanvas').width = window.innerWidth;
```

Comment:
Since this is more of a script than a class, leaving the file name as starting with a lowercase letter, unlike the other files